### PR TITLE
new test_ouput and  solution folder in TMP_WORK

### PR DESCRIPTION
**Pull request by drumilpatel2000**
Refactoring code as per :- https://docs.google.com/document/d/12Vir7w7Vyhz03NvjHyIXbGsXXnFKrK__N1rK3iSUhR8/edit#heading=h.6u3dc964pwj2
This PR adds two new folders in TMP_WORK 
1. test_output
2. solution

### CODE REVIEW

**Comment by emaicus** [comment]
Read through code and all appears in order. Will test tomorrow morning.

**Comment by bmcutler** [comment]
Hi @drumilpatel2000 Please don't refer to the private google docs in your PR message.  Please write a brief summary directly in the PR message so the information is publicly accessible.  Thanks

File: autograder/autograder/grade_item.py [hunk 1]
```diff
@@ -252,6 +252,7 @@ def grade_from_zip(my_autograding_zip_file,my_submission_zip_file,which_untruste
     checkout_path = os.path.join(tmp_submission, "checkout")
 
     provided_code_path = os.path.join(tmp_autograding, "provided_code")
+    solution_path = os.path.join(tmp_autograding, "solution")
```
Comment (bmcutler) [inline autograder/autograder/grade_item.py]: let's rename this folder instructor_solution
(So it's more obvious who's solution it is)

Comment (drumilpatel2000) [reply autograder/autograder/grade_item.py]: @bmcutler Can you review it again?

**Pull request merged**